src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -852,6 +857,45 @@ ApiGlobe.prototype.removeEventListenerLayerChanged = function removeEventListene
     this.viewerDiv.removeEventListener('layerchanged:index', this.callbackLayerChanged, false);
 };
 
+/**
+ * Add a control.
+ * @constructor
+ * @param {object} Control - The Control object.
+ */
+
+ApiGlobe.prototype.addControl = function addControl(control) {
+    var index = this.controls.indexOf(control);
```

Comment:
You're probably right, didn't know this method at the time. Will change that. Or Maybe a `new Map()` can do the job too.